bin/cli.js

```diff
@@ -55,15 +55,25 @@ opts.nonmembers = opts.nonmembers ? opts.nonmembers.split(',') : [];
 missedIssues(opts)
   .then(issues => {
     var lastRepo = null;
+    var lastDate = null;
     issues
       .sort((a, b) => {
-        if (a.repo === b.repo) return a.id - b.id;
+        if (a.repo === b.repo){
+          if (a.updated_at === b.updated_at)
+            return a.id - b.id;
+          return a.updated_at - b.updated_at;
+        }
         return a.repo < b.repo ? -1 : 1;
       })
       .map(iss => {
         if (lastRepo !== iss.repo) {
           console.log(`${lastRepo === null ? '' : '\n'}## ${iss.repo}\n`); // eslint-disable-line no-console
           lastRepo = iss.repo;
+          lastDate = null;
+        }
+        if (lastDate !== iss.updated_at){
+          console.log(`\n**${iss.updated_at}**\n`); // eslint-disable-line no-console
```

Comment:
Currently this prints out `undefined` as the variable is `updated` rather than `updated_at`